packages/jest-worker/README.md

```diff
@@ -39,6 +39,12 @@ export function hello(param) {
 }
 ```
 
+## Experimental worker
+
+Node 10 shipped with [worker-threads](https://nodejs.org/api/worker_threads.html), a "threading API" that uses SharedArrayBuffers to communicate between the main process and its child threads. This experimental Node feature can significantly improve the communication time between parent and child processes in `jest-worker`.
+
+We will use worker threads where available. To enable in Node 10+, run the Node process with the `--experimental-worker` flag.
```

Comment:
Should it be an explicit option? At least to `jest-worker`, but maybe default within Jest?

Reply:
wdym? this is a node option not a jest option

Reply:
I mean run your `canUseWorkerThreads` function inside of `jest` (`jest-runner`? whatever spawns the workers) and pass `useWorkerThreads: boolean` to `jest-worker`. That way consumers of `jest-workers` can choose themselves if they want to use worker threads or not

Reply:
I think in the previous PR we said if worker threads are available we want to use them (instead of opting into them)

Reply:
Aye, for Jest itself. `jest-worker` is useful outside of Jest though, and it feels like an opinionated choice. Maybe not though, it is natural for `jest-worker` to use worker threads

Reply:
I think if someone wants the option they will ask us to enable it and can explain the use case

Reply:
I'm in favor of hiding the complexity associated to a particular strategy if possible. We already give the choice to inject whatever the user prefers if they have a choice. We also have to properly document the new options that can be passed now.